### DISCUSSION TITLE
How do I get CTRL+P / CTRL+N non-IntelliSense completion identical to how it works in VIM?

### DISCUSSION THREAD

**Discussion by danielemegna** [VSCodeVim/Vim#7612] [Q&A]
The main thing missing in my VSCodeVim setup is CTRL+P and CTRL+N simple word completion when in insert mode.
I have googled extensively on how to set this up, but found nothing that works the way I expect.

I do not want IntelliSense to interfere with this completion, I just want simple word completion that works in any context (e.g., comments, strings, etc). Here is an example:

```c
int a_very_long_variable_name;

// a_ve
```

When I've typed `a_ve` in the comment, CTRL+P should suggest `a_very_long_variable_name` (as well as any other word in the document starting with a_ve). CTRL+N should do the same, except that suggestions should be ordered differently.

Is it possible to get this behavior in VSCode with VSCodeVim?

**Comment by higoswami** [comment]
Add this to `settings.json` . 

```    
    "editor.suggestSelection": "recentlyUsed",
    "editor.quickSuggestions": {

        "other": "on",
        "comments": "on",
        "strings": "on"
    },
```

It wont be exactly what you want but you can give it a try
